Podfile

```diff
@@ -187,7 +187,7 @@ target 'WordPress' do
     # While in PR
     #pod 'WordPressAuthenticator', :git => 'https://github.com/wordpress-mobile/WordPressAuthenticator-iOS.git', :branch => ''
```

Comment:
This isn't using the associated Auth branch. I suspect this should be pointed to `issue/224-remove-showSignupMethod`?

Reply:
Also, when I did that locally, I got:

```
[!] CocoaPods could not find compatible versions for pod "WordPressKit":
  In snapshot (Podfile.lock):
    WordPressKit (= 4.6.0, ~> 4.6.0)

  In Podfile:
    WordPressAuthenticator (from `https://github.com/wordpress-mobile/WordPressAuthenticator-iOS.git`, branch `issue/224-remove-showSignupMethod`) was resolved to 1.12.0-beta.4, which depends on
      WordPressKit (~> 4.7.0-beta.1)

    WordPressKit (~> 4.6.0)

Specs satisfying the `WordPressKit (~> 4.6.0), WordPressKit (= 4.6.0, ~> 4.6.0), WordPressKit (~> 4.7.0-beta.1)` dependency were found, but they required a higher minimum deployment target.
```

Reply:
omg haha. That's what I get for trying to juggle 3 separate PRs for Authenticator 😄